docs/source/advanced/io.rst

```diff
@@ -48,7 +48,7 @@ descriptor, so another way to save would be::
 
 .. note::
     The advantage of using a filename instead of file handle is that data will be written to a temporary file
-    first and temporary file will be renamed to provided filename only after all data has been written.
+    first and a temporary file will be renamed to provided filename only after all data has been written.
```

Comment:
Maybe `and the temporary file` since it's the same file?

Reply:
Good point! That definitely sounds smoother. Fixed it :)